docs/4.2/admin-guide.md

```diff
@@ -1956,6 +1956,16 @@ To allow access for all users:
   + Copy `teleport-user-ca.pub` to `/etc/ssh/teleport-user-ca.pub`
   + Update `sshd` configuration (usually `/etc/ssh/sshd_config` ) to point to
     this file: `TrustedUserCAKeys /etc/ssh/teleport-user-ca.pub`
+ 
+ To connect to the OpenSSH server via `tsh` add the `--port=<ssh port>` with the `tsh ssh` command:
+  + Example ssh to `database.work.example.com` as `root` with a OpenSSH server on port 22 via `tsh`
+ ```bash
+    tsh ssh --port=22 root@database.work.example.com
+ ```
+    
+!!! warning "Warning"
+
+    The user being connected as must be available in the Teleport user configuration or role.
```

Comment:
The principal (username) being used to connect must be listed in the Teleport user/role configuration.

(bit more readable, plus uses the term 'principal' which people could potentially Ctrl+F for if they're getting the "name is not a listed principal" error)